### ISSUE TITLE
Multi-language tag not supported in all descriptions

### ISSUE THREAD

**Issue by nantas** [fireball-x/firedoc#6]
Here's a checklist of multi-language tag support in different tag's descriptions:
- [x] module
- [x] module/main
- [x] module property
- [x] module method
- [x] module method param
- [x] class
- [x] class property
- [x] class method
- [x] class method param
- [x] enum
- [x] enum property

Updated based on test for v0.8.26

**Comment by yorkie** [comment]
This should be fixed by 61535b932c976e55674d1e728dbd3da62c1010ac and `firedoc@0.8.10` seems to be working with all of the `description` of the following tags:
- `class` and `enum`
- `param`
- `property`, `attribute`, `event`, `method` and `config`

@nantas could you please make a double check for the list, thanks :p

**Comment by nantas** [comment]
tested on v0.8.17, now only module description is not working.

**Comment by yorkie** [comment]
Should be fixed at `firedoc@0.8.18`

**Comment by nantas** [comment]
it works on other modules, except Fire, which declared as: 

``` js
    /**
     * !#en
     * Global object with runtime classes, properties and methods you can access from anywhere.
     * Submodules:
     * - [JS](./Fire.JS.html)
     * - [Spine](./Fire.Spine.html)
     *
     * !#zh
     * 可全局访问的公共方法和属性，也会包括一些组件和类的静态方法
     * 包含的子模块:
     * - [JS](./Fire.JS.html)
     * - [Spine](./Fire.Spine.html)
     *
     * @module Fire
     * @main Fire
     */
```

**Comment by yorkie** [comment]
This bug can be reproduced by using `@main` tag and now fixed at b703709b3e2613196da492f570826e45107ec34e.

**Comment by nantas** [comment]
all clear now
